src/js/BookReader/Mode1Up.js

```diff
@@ -226,38 +304,39 @@ export class Mode1Up {
       // current index in drawLeafsOnePage after we create the new view container
 
       // Make sure this will count as current page after resize
-      const fudgeFactor = (this.book.getPageHeight(this.br.currentIndex()) / this.br.reduce) * 0.6;
+      const fudgeFactor = 0.6 * this.physicalInchesToDisplayPixels(this.book.getPage(this.br.currentIndex()).heightInches);
       const oldLeafTop = this.getPageTop(this.br.currentIndex()) + fudgeFactor;
-      const oldViewDimensions = this.calculateViewDimensions(this.br.reduce, this.br.padding);
+      const oldViewDimensions = this.calculateViewDimensions();
       scrollRatio = oldLeafTop / oldViewDimensions.height;
     }
 
     // Recalculate 1up reduction factors
     this.calculateReductionFactors();
     // Update current reduce (if in autofit)
     if (this.br.onePage.autofit) {
-      const reductionFactor = this.br.nextReduce(this.br.reduce, this.br.onePage.autofit, this.br.onePage.reductionFactors);
-      this.br.reduce = reductionFactor.reduce;
+      const reductionFactor = this.br.nextReduce(this.realWorldReduce, this.br.onePage.autofit, this.br.onePage.reductionFactors);
+      this.realWorldReduce = reductionFactor.reduce;
     }
 
-    const viewDimensions = this.calculateViewDimensions(this.br.reduce, this.br.padding);
-
-    this.br.refs.$brPageViewEl.height(viewDimensions.height);
-    this.br.refs.$brPageViewEl.width(viewDimensions.width);
+    const viewDimensions = this.calculateViewDimensions();
+    this.$documentContainer.height(viewDimensions.height);
+    this.$documentContainer.width(viewDimensions.width);
 
+    // Remove all pages
+    this.$documentContainer.empty();
+    this.br.displayedIndices = [];
 
-    const newCenterY = scrollRatio * viewDimensions.height;
-    const newTop = Math.max(0, Math.floor( newCenterY - this.br.refs.$brContainer.height() / 2 ));
-    this.br.refs.$brContainer.prop('scrollTop', newTop);
-
-    // We use clientWidth here to avoid miscalculating due to scroll bar
+    // Scroll to the right spot
     const newCenterX = oldCenterX * (viewWidth / oldPageViewWidth);
-    let newLeft = newCenterX - this.br.refs.$brContainer.prop('clientWidth') / 2;
-    newLeft = Math.max(newLeft, 0);
-    this.br.refs.$brContainer.prop('scrollLeft', newLeft);
+    const newCenterY = scrollRatio * viewDimensions.height;
```

Comment:
I tend to prefer object destructure, but I have seen the reference not reflect the "fresh scope" some of these function instances hold inside the core code...I am wondering if you have seen that as well?
if not, what should be our baseline style if scope isn't an option - destructure or traversal like `viewDimensions.height`?

Reply:
Hmm, not sure I follow the scope thing. Can you give an example?

For readability here, I think `viewDimensions.height` is more readable, cause if I destructure to just `height`, there are like 6 different heights at play in this block of code :D

Reply:
> Hmm, not sure I follow the scope thing. Can you give an example?
this was an overall observation, esp. working with lit element, i've found that `const { fooFn } = this` can break fooFn scope. so anytime something gets/doesn't get destructured, my feelers go up.

readability-wise, the object traversal is for sure more helpful ^_^